### PR TITLE
Centralize advanced settings

### PR DESCRIPTION
**Pull request by pannal**
GHI (If applicable): #

## Description:
Centralizes advanced (non-Plex) settings in one place

## Checklist:
- [x] I have based this PR against the develop branch

### CODE REVIEW

File: lib/util.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ def watchStatusChanged(self):
     def onNotification(self, sender, method, data):
         if sender == 'script.plex' and method.endswith('RESTORE'):
             from windows import kodigui
+            getAdvancedSettings()
```
Comment (ruuk) [inline lib/util.py]: Do we really need this? Isn't it already added globally with `advancedSettings = AdvancedSettings()` right after the class definition.

Comment (pannal) [reply lib/util.py]: Yes, but if the user exits p4k by minimizing it, then changes the addon settings and then reactivates p4k, the changes won't be reflected.

Comment (ruuk) [reply lib/util.py]: Ah, yeah. Good call!

**Comment by ruuk** [comment]
Looks good!

**Pull request merged**